setup.py

```diff
@@ -180,6 +180,7 @@ def run(self):
         extras_require={
             "onnx": ["numpy", "onnx", "onnx-runtime"],
             "ipex": ["intel_extension_for_pytorch"],
+            "openvino": ["openvino"],
```

Comment:
I'm not sure if we should be doing this.
Increasing size of docker images is a concern and we don't want to add to the list.
 
We want to follow what we did with TensorRT where the user needs to include tensorrt in requirements.txt. Can you please change this. https://github.com/pytorch/serve/blob/master/examples/torch_tensorrt/README.md

Reply:
Made the change